build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
 apply plugin: 'cam72cam.universalmodcore'
 apply plugin: 'maven'
 
-String version = '1.0.2'
+String version = '1.0.3'
```

Comment:
There is no need to change the version here anymore. It's handled by the release pipeline